embeddings/task/lightning_task/lightning_task.py

```diff
@@ -86,14 +98,52 @@ def fit_predict(
         self,
         data: HuggingFaceDataModule,
         predict_subset: LightingDataModuleSubset = LightingDataModuleSubset.TEST,
+        run_name: Optional[str] = None,
     ) -> Dict[str, nptyping.NDArray[Any]]:
         if not self.model:
             raise self.MODEL_UNDEFINED_EXCEPTION
-        self.fit(data)
+        self.fit(data, run_name=run_name)
         dataloader = data.get_subset(subset=predict_subset)
         assert isinstance(dataloader, DataLoader)
-        return self.predict(dataloader=dataloader)
+        result = self.predict(dataloader=dataloader)
+        return result
 
     @abc.abstractmethod
     def build_task_model(self) -> None:
         pass
+
+    def get_lightning_loggers(
```

Comment:
Since we now have a logging wrapper I'd suggest moving this method from Task to one of the Logger/LoggingConfig